layout/src/Layout.ts

```diff
@@ -224,7 +224,10 @@ export class Layout extends LitElement {
   }
 
   private onNavigationWrapperClick(event: Event): void {
-    if (event.currentTarget === event.target) {
+    if (
+      event.target === event.currentTarget ||
+      (event.target as HTMLElement).tagName === 'A'
```

Comment:
TS has weird types for `event.target`, not the first time encountering the need to write such hacks
anyone having idea why is it so? and any way to do it better?